### PR TITLE
Readme clean up

### PR DESCRIPTION
**Pull request by camilamaia**
Readme clean up. Move documentation from README.md to the website

Closes https://github.com/scanapi/scanapi/issues/227

### CODE REVIEW

**Comment by abreumatheus** [comment]
@camilamaia I think maybe you should leave the documentation section in the file and in the table of contents, just for the ease of access. Otherwise, the user will not know where to find the link to the documentation until he scrolls all the way down.
Ex:
```
# Documentation
The full documentation is available at [scanapi.dev](https://scanapi.dev)!
```

File: README.md [hunk 1]
```diff
@@ -102,240 +91,7 @@ report results.
...
-
-```yaml
-report:
-  hide-response:
-    headers:
-      - Authorization
-```
-
-Available attributes to hide: `headers`, `body` and `url`.
+The full documentation is available at [scanapi.dev][https://scanapi.dev]!
```
Comment (abreumatheus) [inline README.md:94]: The hyperlink should be enclosed in parenthesis here :)
Ex:
`The full documentation is available at [scanapi.dev](https://scanapi.dev)!`

**Comment by camilamaia** [comment]
> @camilamaia I think maybe you should leave the documentation section in the file and in the table of contents, just for the ease of access. Otherwise, the user will not know where to find the link to the documentation until he scrolls all the way down.
> Ex:
> 
> ```
> # Documentation
> The full documentation is available at [scanapi.dev](https://scanapi.dev)!
> ```

great idea! I am going to do the same for examples. Thanks!

**Comment by camilamaia** [comment]
@abreumatheus done! ✅

**Pull request merged**